src/core/Akka.Remote/Configuration/Remote.conf

```diff
@@ -524,48 +521,45 @@ akka {
+          #     default-key-set | exportable | machine-key-set | persist-key-set | user-key-set | user-protected
+          # flags = [ "default-key-set" ]
+          
+          # To use a Thumbprint instead of a file, set this to true
+          # And specify a thumbprint and it's storage location below
+          use-thumbprint-over-file = false
+          
+          # Valid Thumbprint required (if use-thumbprint-over-file is true)
+          # A typical thumbprint is a format similar to: "45df32e258c92a7abf6c112e54912ab15bbb9eb0"
+          # On Windows machines, The thumbprint for an installed certificate can be located
+          # By using certlm.msc and opening the certificate under the 'Details' tab.
+          thumbprint = ""
+          
+          # The Store name. Under windows The most common option is "My", which indicates the personal store.
+          # See System.Security.Cryptography.X509Certificates.StoreName for other common values.
+          store-name = ""
+          
+          # Valid options : local-machine or current-user
+          # current-user indicates a certificate stored under the user's account
+          # local-machine indicates a certificate stored at an operating system level (potentially shared by users)
+          store-location = "current-user"
+        }
+        suppress-validation = false
+      }
     }
 
     dot-netty.udp = ${akka.remote.dot-netty.tcp}
     dot-netty.udp {
       transport-protocol = udp
     }
 
-    dot-netty.ssl = ${akka.remote.dot-netty.tcp}
-    dot-netty.ssl = {
-		certificate {
-			# Valid certificate path required
-			path = ""
-			# Valid certificate password required
-			password = ""
-			# Default key storage flag is "default-key-set"
-			# Available flags include: 
-			#     default-key-set | exportable | machine-key-set | persist-key-set | user-key-set | user-protected
-			# flags = [ "default-key-set" ]
-
-			# To use a Thumbprint instead of a file, set this to true
-			# And specify a thumprint and it's storage location below
-			use-thumbprint-over-file = false
-
-			# Valid Thumprint required (if use-thumbprint-over-file is true)
-			# A typical thumbprint is a format similar to: "45df32e258c92a7abf6c112e54912ab15bbb9eb0"
-			# On Windows machines, The thumprint for an installed certificate can be located
-			# By using certlm.msc and opening the certificate under the 'Details' tab.
-			thumbprint = ""
-
-			# The Store name. Under windows The most common option is "My", which indicates the personal store.
-			# See System.Security.Cryptography.X509Certificates.StoreName for other common values.
-			store-name = ""
-
-			# Valid options : local-machine or current-user
-			# current-user indicates a certificate stored under the user's account
-			# local-machine indicates a certificate stored at an operating system level (potentially shared by users)
-			store-location = "current-user"
-		}
-		suppress-validation = false
-    }
-
```

Comment:
We're not following JVM SSL convention, this HOCON path has not been used for the last 5 years